Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GIT
   remote: git@github.com:wordpress-mobile/release-toolkit
-  revision: eccf5a273276e976cd42c66590af087d1654037f
-  branch: develop
+  revision: 83e4d5f036f1871ce6f8e9411550d4040cebddca
+  branch: add/github-comment-action
```

Comment:
Asking because the PR is "Ready for review", not the "Draft": do you plan on shipping a new release toolkit build before merging this?

IMHO it would be appropriate, but I'm okay if it doesn't happen.

Reply:
Just chiming in to say that I will still need some time to finish other steps in my L10n work (porting widgets etc) before I need to get back to edit the release-toolkit to port more actions to it. Which means that I'm ok with us doing a minor version of the release-toolkit for this if deemed appropriate indeed.

That is, as long as @mokagio you confirm that you fixing your local env on https://github.com/wordpress-mobile/release-toolkit/issues/314#issuecomment-982860291 and https://github.com/wordpress-mobile/release-toolkit/issues/315#issuecomment-982884562 fixed those issues you had and that those are not blockers to making a new version which would include localization changes too.

PS: I'll make a quick PR to isolate my change that fixes #314 so it can be merged easier (rather than waiting for my PR on next widget to land)

Reply:
I'm following @AliSoftware's lead on this, so I'm fine to continue to target `develop` (I'll update this reference) as long as he needs to, or move to a tag if one is created.

Reply:
I followed up on the `dirname` issue [confirming](https://github.com/wordpress-mobile/release-toolkit/issues/315#issuecomment-982884562) it was due to messed up state on my end. Still unclear about the `--tags` one, but https://github.com/wordpress-mobile/WordPress-iOS/pull/17627 will prevent it from happening during a code freeze again (whatever the tags-related cause is), so we're good 👍